package.json

```diff
@@ -423,14 +422,26 @@
     "@types/slate": "0.47.11",
     "@rushstack/rig-package": "0.5.1",
     "@rushstack/ts-command-line": "4.16.1",
-    "ngtemplate-loader/loader-utils": "^2.0.0",
     "trim": "1.0.1",
     "semver@~7.0.0": "7.5.4",
     "semver@7.3.4": "7.5.4",
     "slate-dev-environment@^0.2.2": "patch:slate-dev-environment@npm:0.2.5#.yarn/patches/slate-dev-environment-npm-0.2.5-9aeb7da7b5.patch",
     "react-split-pane@0.1.92": "patch:react-split-pane@npm:0.1.92#.yarn/patches/react-split-pane-npm-0.1.92-93dbf51dff.patch",
     "@storybook/blocks@7.4.5": "patch:@storybook/blocks@npm%3A7.4.5#./.yarn/patches/@storybook-blocks-npm-7.4.5-5a2374564a.patch"
   },
+  "pnpm": {
+    "overrides": {
+      "ngtemplate-loader>loader-utils": "^2.0.0",
+      "slate-dev-environment@^0.2.2": "^0.2.2",
+      "react-split-pane@0.1.92": "0.1.92",
+      "@storybook/blocks@7.4.5": "7.4.5"
```

Comment:
These overrides are required to "undo" the yarn resolutions. The patches are applied by different configuration.

If we switch fully to pnpm, we could just remove these from both the resolutions and pnpm.overrides objects.